app/serializers/rest/quote_serializer.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class REST::QuoteSerializer < ActiveModel::Serializer
+  attributes :state
+
+  has_one :quoted_status, serializer: REST::ShallowStatusSerializer
+
+  def quoted_status
```

Comment:
Some of this could be extracted to a presenter or model method to simplify logic in the two serializers.